filters/include/pcl/filters/experimental/transform_filter.h

```diff
@@ -14,6 +14,8 @@
 
 #include <boost/optional.hpp> // std::optional for C++17
 
+#include <boost/optional.hpp> // std::optional for C++17
```

Comment:
Optional a second time is optional (not needed)

Reply:
Done, it was somehow duplicated during rebase